packages/mui-material/src/ListItemText/ListItemText.js

```diff
@@ -93,29 +96,49 @@ const ListItemText = React.forwardRef(function ListItemText(inProps, ref) {
 
   const classes = useUtilityClasses(ownerState);
 
+  const externalForwardedProps = {
+    slots,
+    slotProps: {
+      primary: primaryTypographyProps,
+      secondary: secondaryTypographyProps,
+      ...slotProps,
+    },
+  };
+
+  const [PrimarySlot, primarySlotProps] = useSlot('primary', {
+    elementType: Typography,
+    externalForwardedProps,
+    ownerState,
+  });
+  const [SecondarySlot, secondarySlotProps] = useSlot('secondary', {
+    elementType: Typography,
+    externalForwardedProps,
+    ownerState,
+  });
```

Comment:
```suggestion
  const [PrimarySlot, primarySlotProps] = useSlot('primary', {
    className: classes.primary,
    elementType: Typography,
    externalForwardedProps,
    ownerState,
  });
  const [SecondarySlot, secondarySlotProps] = useSlot('secondary', {
    className: classes.secondary,
    elementType: Typography,
    externalForwardedProps,
    ownerState,
  });
```
className should be added to `useSlot` so that it's merged with the incoming `slotProps`.